README.md

```diff
@@ -30,7 +28,7 @@ For these libraries, we accept bug reports and pull requests. The corresponding
 
 **ODataV4-7.x branch**
 
-The ODataV4-7.x branch includes the .NET libraries for OData V4 only that are in development now. It is evolved from ODataV4-6.x with kinds of improvements, new features and so on. It is in-compatible with ODataV4-6.x release and will become the master branch after it is released. It has same libraries as ODataV4-6.x branch.
+The ODataV4-7.x branch includes .NET Standard 1.1 libraries for OData V4 only that are in development now. This branch builds upon ODataV4-6.x with new features with the same libraries but is incompatible with ODataV4-6.x--it will become the master branch after it is released. Due to the new use of `project.json`, there is a change in the build process for those who are unfamiliar. Please either 1) run `nuget restore Microsoft.Test.OData.DotNetStandard.sln` in the commandline of the `odata.net/sln` directory, or 2) enable automatic package restore via Visual Studio to build them (in VS2015, go to `Tools->NuGet Package Manager->Package Manager Settings`. Enable "Allow NuGet to download missing packages".). More info regarding the command can be found [here](https://docs.microsoft.com/en-us/nuget/consume-packages/package-restore).
```

Comment:
Need to revise statement. It can build both PCL and .NET Standard. Also rewrite some sentences for easier understanding.